src/TimeoutMigrationTool/MigrationRunner.cs

```diff
@@ -13,13 +13,39 @@ public MigrationRunner(ITimeoutStorage timeoutStorage, ICreateTransportTimeouts
             this.transportTimeoutsCreator = transportTimeoutsCreator;
         }
 
-        public async Task Run(Dictionary<string, string> runParameters)
+        public async Task Run(IDictionary<string, string> runParameters)
         {
-            var toolState = await timeoutStorage.GetOrCreateToolState().ConfigureAwait(false);
+            var forceMigration = runParameters.ContainsKey(ApplicationOptions.ForceMigration);
+            //if (forceMigration) 
```

Comment:
I'd say if we move this into the if-statement on line 26 we can write "Existing incomplete migration will be aborted" or something along those lines?